app/templates/activity.hbs

```diff
@@ -5,4 +5,6 @@
 {{activity-script-viewer activity=model}}
 {{activity-error-counter activity=model}}
 
-{{outlet}}
+{{blureable-button action="restart" text="Restart activity" cssClass="btn-restart btn-primary"}}
```

Comment:
You can use `classNames=""` directly. The css clases you define in that attribute will be appended to the ones you define in component's js class